src/common/IsolatedWorld.ts

```diff
@@ -69,6 +69,23 @@ export interface PageBinding {
   pptrFunction: Function;
 }
 
+/**
+ * @internal
+ */
+export const DEFAULT_WORLD = Symbol('defaultWorld');
```

Comment:
should we add some docs for the future us to describe the intention behind the two worlds?